assets/js/components/GoalsSidebar.vue

```diff
@@ -34,7 +33,6 @@ import Results from "./Results";
 export default {
   components: {
     Embeddor,
-    xsummary: Summary,
```

Comment:
I think the linter may cry about the unused import after this is removed? Or maybe not I'm not sure :sweat_smile: